docs/source/tutorial.md

```diff
@@ -54,21 +15,24 @@ For example, see below:
 ```python
```

Comment:
This tutorial is a good start, but perhaps when you come around to editing it again you might describe what the user would want to do, and how they can use your library to achieve that. At the moment I can infer how I'd use `convert()` from the values you specify for the arguments, but having it spelled out explicitly would be nice.

Also, I see that there are quite a few methods that are part of the public interface, but aren't mentioned here in the tutorial. It'd be good to see everything that's public described with a use case, perhaps here in the tutorial, or maybe in the documentation for the method. If there are parts that don't have a use case for the end user, you might consider making them private.

Reply:
Thanks - you bring up great points with this!

- On the tutorial content: I'll plan to revise the tutorial as part of future edits and create an issue from this comment to make sure this is addressed.
- On the public vs private methods: `convert` is the primary means by which one uses this library currently. I think I was somewhat channeling [`Hyrum's Law`](https://www.hyrumslaw.com/), assuming that one may end up using any part of this package given the source. This shouldn't be an excuse from distinguishing from public vs private. I'll make some updates here to address this matter within the scope of this PR.

Reply:
Sounds good, and FWIW I'm all for documenting both "private" and "public" methods, since as you observed someone may come to rely on them.

Later on, while documenting the interface (both public and private) I think it'd be helpful for a reader to know how the methods within the library are composed or could be composed. You could do this by calling out methods that rely on the current method or vice versa in the docs (e.g., "called by X(), used as input to Y()"), to at least demonstrate one example of how they could be composed. If you wanted to take this idea further, you could define named types (https://docs.python.org/3/library/typing.html#type-aliases) that are returned or taken as arguments to the methods. For example, `to_parquet()` takes several arguments whose types are `Union[List[str], Tuple[str, ...]]`; if you were to instead distinguish each argument as a type of the thing you expect it to take, you could simply read the signatures of `to_parquet()` and other methods in the library to see how you'd chain them together.

On that note, you could probably save yourself some typing (no pun intended) by aliasing `Union[List[str], Tuple[str, ...]]` as, say, `IterableCollection`. Not that that's a great name, but as the saying goes, "There are only two hard things in Computer Science: cache invalidation and naming things." (https://www.karlton.org/2017/12/naming-things-hard/). 😛

(I think my comment here would be better implemented down the road in another PR; I don't think it's necessary to get it into this one.)

Reply:
Thanks for the follow up here! I'll further address this topic in a new issue referencing your comments, as you suggested.

Reply:
Initial public/private distinctions have been added via [90db7e3](https://github.com/cytomining/pycytominer-transform/pull/22/commits/90db7e3dd00fe4b07b71d1dc9cc51e71943e9f25). Other topics have been moved into issues to resolve with later PR's. Marking this as resolved, thanks again!